Unit Tests/Tests/Networking/Model/Subscription/PresenceSubscription/Wire.Event.PresenceState+Decodable.Tests.swift

```diff
@@ -1,4 +1,5 @@
 import XCTest
+@testable import TestUtilities
```

Comment:
Does it make any sense to mark `TestUtilities` imports as `@testable`. `TestUtilities` should have its own public interface and we should not relay on internal implementations provided by it.

Reply:
I asked myself the same question.  We can either explicitly mark things (well pretty everything I guess) as `public` in `TestUtilitilies` or just use `@testable`.   

If you'd like me to change to `public` then just say and I'll make it so.

Reply:
I think that `TestUtilities` is a framework that provides us with functionality that should have its own public interface. `@testable` might indicate a module that we are about to test.

Reply:
Completely agree, will change.

Reply:
This now done FYI